bot/build.sh

```diff
@@ -247,7 +247,14 @@ declare -a TARBALL_STEP_ARGS=()
 TARBALL_STEP_ARGS+=("--save" "${TARBALL_TMP_TARBALL_STEP_DIR}")
 
 # determine temporary directory to resume from
-TARBALL_STEP_ARGS+=("--resume" "${REMOVAL_TMPDIR}")
+if [[ -z ${REMOVAL_TMPDIR} ]]; then
+    # no rebuild step was done, so the tarball step should resume from the build directory
+    BUILD_TMPDIR=$(grep ' as tmp directory ' ${build_outerr} | cut -d ' ' -f 2)
+    TARBALL_STEP_ARGS+=("--resume" "${BUILD_TMPDIR}")
```

Comment:
correct me if I'm wrong, but are we sure there is always a tmpdir to resume from?

Reply:
Yes, there should be? The build step is always run, and even if it builds nothing, it contains for instance the CVMFS files.

Reply:
Yes, there is always a tmpdir to resume from ***within*** a job. We also store the tmpdir in a tarball, so we can later use that for inspection. We need to do this because the tmpdir should be cleaned up when the job finishes.